Makefile

```diff
@@ -91,15 +91,18 @@ godocs: ${GO_DOCS_ARTIFACTS}
 
 
 release:
-	git checkout gh-pages || exit
+	git checkout -b build-release-${VERSION} || exit
 	sed -i s/tag\:\ master/tag\:\ ${VERSION}/ chart/values.yaml
 	helm package --version ${VERSION} chart
```

Comment:
For these two lines, the versions could be in Chart.yaml like this:

```
appVersion: 0.1.x
version: 0.1.x
```

In the templates then you can default to {{ .Chart.appVersion }}, eliminating the need to do the `sed`, but still allowing anyone to overwrite that.

```
image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Reply:
The chart version should also be updated in the Chart.yaml file every time there is a change to any of the chart files. So there should be no real need to overwrite it when packaging the chart.

Reply:
Agree that we should add appVersion to the chart.yaml.
Will change values.yaml as suggested when I implement that.
so I would sed the chart.yaml instead of values.yaml to fill in the chart and application version